README.md

```diff
@@ -53,38 +48,147 @@ See the [Docker documentation](./docs/Docker.md) to get up and running
   $ psql -c "CREATE DATABASE upaya_test;"
```

Comment:
For future consideration, we should see if we can integrate the database creation into the `make setup` step, to reduce the number of steps by one.